environment/container/kubernetes/k3s.go

```diff
@@ -122,6 +122,9 @@ func installK3sCluster(
 		"--resolv-conf", "/etc/resolv.conf",
 	}
 
+	// disable traefik defaults
+	args = append(args, "--disable", "traefik")
```

Comment:
this is going to make traefik always disabled

Reply:
yes, this is going to disable traefik always. I revert this commit.